server/src/main/java/org/elasticsearch/index/fielddata/plain/BytesBinaryDVLeafFieldData.java

```diff
@@ -23,19 +23,19 @@
 import org.apache.lucene.store.ByteArrayDataInput;
 import org.apache.lucene.util.Accountable;
 import org.apache.lucene.util.BytesRef;
-import org.elasticsearch.index.fielddata.AtomicFieldData;
+import org.elasticsearch.index.fielddata.LeafFieldData;
 import org.elasticsearch.index.fielddata.ScriptDocValues;
 import org.elasticsearch.index.fielddata.SortedBinaryDocValues;
 
 import java.io.IOException;
 import java.util.Collection;
 import java.util.Collections;
 
-final class BytesBinaryDVAtomicFieldData implements AtomicFieldData {
+final class BytesBinaryDVLeafFieldData implements LeafFieldData {
```

Comment:
nit: `ScriptDocValues.java` in a [comment](https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/index/fielddata/ScriptDocValues.java#L455) still mentions the old name of this class, would be nice to change the comment as well.